eval/scripts/dictionary_eval.py

```
@@ -0,0 +1,147 @@
+    if not baseline_path.exists():
+        raise SystemExit(f"{baseline_path} not found — run run_eval.py --sets {args.set} --models {args.model} first")
+
+    rows = [json.loads(line) for line in open(baseline_path) if "summary" not in json.loads(line)]
+    out_path = RESULTS / f"{args.set}--{args.model}--with-dictionary.jsonl"
+
+    refs, hyps_baseline, hyps_dict = [], [], []
+    clips_with_entries = 0
+    with open(out_path, "w") as out_f:
+        for row in rows:
+            manifest_entry = next(
+                (json.loads(l) for l in open(DATA / args.set / "refs.jsonl") if json.loads(l)["id"] == row["id"]),
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)